build-packages.cmd

```diff
@@ -60,6 +60,15 @@ if NOT [!ERRORLEVEL!]==[0] (
   exit /b 1
 )
 
+rem Build the TestHost package
+set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.TestHost\Microsoft.NETCore.TestHost.builds" !allargs!
```

Comment:
Why does this need to be added directly to this cmd/sh script? I would expect them to just call build on packages.builds file.